Author: Jeroen Schot <schot@a-eskwadraat.nl>
Date: Fri Apr 18 11:27:21 CEST 2014
Description: Fix man page formatting
 This patch solves two minor issues in the extension
 man pages:
 - ... is used as a comment marker.
 - Some hyphens where a minus sign is intended.

--- gawk-4.1.0.h+dfsg.orig/extension/filefuncs.3am
+++ gawk-4.1.0.h+dfsg/extension/filefuncs.3am
@@ -148,7 +148,7 @@ if this field is present on your system.
 \fBstatdata["pmode"]\fP
 A human-readable version of the mode value, such as printed by
 .IR ls (1).
-For example, \fB"-rwxr-xr-x"\fP.
+For example, \fB"\-rwxr\-xr\-x"\fP.
 .TP
 \fBstatdata["linkval"]\fP
 If the named file is a symbolic link, this element will exist
--- gawk-4.1.0.h+dfsg.orig/extension/fnmatch.3am
+++ gawk-4.1.0.h+dfsg/extension/fnmatch.3am
@@ -73,7 +73,7 @@ Nothing prevents AWK code from changing
 variabale
 .BR FNM_NOMATCH ,
 but doing so may cause strange results.
-... .SH BUGS
+\# .SH BUGS
 .SH EXAMPLE
 .ft CW
 .nf
--- gawk-4.1.0.h+dfsg.orig/extension/fork.3am
+++ gawk-4.1.0.h+dfsg/extension/fork.3am
@@ -36,7 +36,7 @@ This function waits for the first child
 The return value is that of the
 .IR wait (2)
 system call.
-... .SH NOTES
+\# .SH NOTES
 .SH BUGS
 There is no corresponding
 .B exec()
--- gawk-4.1.0.h+dfsg.orig/extension/inplace.3am
+++ gawk-4.1.0.h+dfsg/extension/inplace.3am
@@ -44,14 +44,14 @@ rule or on the command line, then the
 extension concatenates that suffix onto the original
 filename and uses the result as a filename for renaming
 the original.
-... .SH NOTES
-... .SH BUGS
+\# .SH NOTES
+\# .SH BUGS
 .SH EXAMPLE
 .ft CW
 .nf
-gawk -i inplace '\f(CIscript\fP' files ...
+gawk \-i inplace '\f(CIscript\fP' files ...
 .br
-gawk -i inplace -f \f(CIscriptfile\fP files ...
+gawk \-i inplace \-f \f(CIscriptfile\fP files ...
 .fi
 .ft R
 .SH "SEE ALSO"
--- gawk-4.1.0.h+dfsg.orig/extension/ordchr.3am
+++ gawk-4.1.0.h+dfsg/extension/ordchr.3am
@@ -28,8 +28,8 @@ whose first character is that represente
 .PP
 These functions are inspired by the Pascal language functions
 of the same name.
-... .SH NOTES
-... .SH BUGS
+\# .SH NOTES
+\# .SH BUGS
 .SH EXAMPLE
 .ft CW
 .nf
--- gawk-4.1.0.h+dfsg.orig/extension/readdir.3am
+++ gawk-4.1.0.h+dfsg/extension/readdir.3am
@@ -52,7 +52,7 @@ You can use the
 extension to call
 .I stat()
 in order to get correct type information.
-... .SH BUGS
+\# .SH BUGS
 .SH EXAMPLE
 .ft CW
 .nf
--- gawk-4.1.0.h+dfsg.orig/extension/readfile.3am
+++ gawk-4.1.0.h+dfsg/extension/readfile.3am
@@ -34,8 +34,8 @@ PROCINFO["readfile"]
 exists.
 When activated, each input file is returned in its entirety as \f(CW$0\fR.
 \f(CWRT\fP is set to the null string.
-... .SH NOTES
-... .SH BUGS
+\# .SH NOTES
+\# .SH BUGS
 .SH EXAMPLE
 .ft CW
 .nf
--- gawk-4.1.0.h+dfsg.orig/extension/revoutput.3am
+++ gawk-4.1.0.h+dfsg/extension/revoutput.3am
@@ -15,7 +15,7 @@ adds a simple output wrapper that revers
 line.
 It's main purpose is to show how to write an output wrapper, although
 it may be mildy amusing for the unwary.
-... .SH BUGS
+\# .SH BUGS
 .SH EXAMPLE
 .ft CW
 .nf
--- gawk-4.1.0.h+dfsg.orig/extension/revtwoway.3am
+++ gawk-4.1.0.h+dfsg/extension/revtwoway.3am
@@ -23,7 +23,7 @@ adds a simple two-way processor that rev
 in each line sent to it for reading back by the AWK program.
 It's main purpose is to show how to write a two-way extension, although
 it may also be mildy amusing.
-... .SH BUGS
+\# .SH BUGS
 .SH "SEE ALSO"
 .IR "GAWK: Effective AWK Programming" ,
 .IR filefuncs (3am),
--- gawk-4.1.0.h+dfsg.orig/extension/rwarray.3am
+++ gawk-4.1.0.h+dfsg/extension/rwarray.3am
@@ -53,7 +53,7 @@ However, double precision floating-point
 native binary data.  Thus, arrays containing only string data
 can theoretically be dumped on systems with one byte order and
 restored on systems with a different one, but this has not been tried.
-... .SH BUGS
+\# .SH BUGS
 .SH EXAMPLE
 .ft CW
 .nf
--- gawk-4.1.0.h+dfsg.orig/extension/time.3am
+++ gawk-4.1.0.h+dfsg/extension/time.3am
@@ -35,8 +35,8 @@ then it returns \-1 and sets
 .BR ERRNO .
 Otherwise, the function should return 0 after sleeping
 for the indicated amount of time.
-... .SH NOTES
-... .SH BUGS
+\# .SH NOTES
+\# .SH BUGS
 .SH EXAMPLE
 .ft CW
 .nf
